tools/cbmc/proofs/DNS/DNSlookup/Makefile.json

```diff
@@ -0,0 +1,23 @@
+{
+  "ENTRY": "DNSlookup",
+  ################################################################
+  # This configuration flag uses DNS cache
+  "USE_CACHE":1,
+  "CBMCFLAGS": 
+  [
+  		"--unwind 1",
+  		"--unwindset prvProcessDNSCache.0:5,strcmp.0:11",
```

Comment:
Is the strcmp unwind limit 11 because it's one more than the length of `pcHostName`?

If so, please define a variable to use both in the Makefile and the C file to avoid duplicating numbers.

```json
"MAX_HOSTNAME_LEN": 10,
"HOSTNAME_UNWIND": "__eval {MAX_HOSTNAME_LEN} + 1",
...
"--unwindset prvCreateDNSMessage.0:{HOSTNAME_UNWIND}",
...
"DEF":
[
  "MAX_HOSTNAME_LEN={MAX_HOSTNAME_LEN}"
]
```
```c
char *pcHostName = malloc(MAX_HOSTNAME_LEN);
```